### ISSUE TITLE
As a user, I want to know what happens after certifying my audit

### ISSUE THREAD

**Issue by melchoyce** [GSA-TTS/FAC#550]
Breakout issue of #539 and related to #538.

This issue covers the design of a new confirmation state for auditors and auditees that would show after they submit their audit certifications and their final audit submissions.

### Background

In our auditee research, we heard that auditees are often unsure what happens next after certifying their audits, and after submitting their audits. We should explore some sort of confirmation, whether as a new page or notice or something else, that informs users more about where their submission is within the single audit submission process.

### Acceptance criteria (We'll know we're done when...)

- [ ] We provide a way for auditors and auditees to know what to do/what comes next next in the submission process.
  
### Tasks

- [ ] Design
- [ ] Peer review
- [ ] Write implementation notes

### Definition of Done

- [ ] Screens in Figma are ready to be handed off to front end
- [x] ~All text has a contrast ratio of 4.5:1 with the background~ (Built into USWDS)
- [ ] Interaction are documented if they deviate from USWDS components
- [ ] The needs of someone navigating the site with the use of a screenreader or keyboard has been considered and documented 
- [x] ~There is an option for multilingual support~ (We've established multilingual support in the header)
- [ ] There is supporting documentation for alt text, labels for links (anchor + target), error text in forms, tab order, focus order, and what assistive technologies should announce (if applicable)

**Comment by pkigongo** [comment]
Potentially blocked by #536

**Comment by austinhernandez** [comment]
This ticket was only to show what happens after certifying an audit, but there is a need to show some kind of alert after the completion of each step and a notification of what's to come. It's tied in with #536 and #539 which is more explicit about whats next and lets people know there will be email alerts. 

I'm trying to solve this problem with modals and temporary alert messages. They need content help! Here are 2 examples: 

![image](https://user-images.githubusercontent.com/23194720/205383728-a3b2d537-2f97-4739-9d68-776a7b6ec06a.png)

When a user receives a completion alert when:

- A submission has been created
- The SF-SAC has been completed
- The audit report has been uploaded
- The auditee has certified
- The auditor has certified

Example:

![image](https://user-images.githubusercontent.com/23194720/205384639-4d36d3eb-63aa-4318-9612-2b0109f742d9.png)

**Comment by austinhernandez** [comment]
These need mega content help. These are all linked in the clickable Figma prototype here: https://www.figma.com/file/1bNncVt1TcFFDhQslqFC2o/FAC-Playground?node-id=5281%3A32325&t=69QiLI2KorGnt7g6-1

Implementation notes. 
- After a section is completed, a success modal appears.
- When a user navigates back to the "report home" page a success alert appears at the top instructing them what needs to be done next. 
- This success message will disappear after a user makes any action to navigate away from the page

**Comment by melchoyce** [comment]
Checking out the modals:

![image](https://user-images.githubusercontent.com/2846578/207435931-1fcf23b5-3bce-43cc-82bd-fb7ecf143571.png)

I'm wondering which we think the primary action here should be, staying in the report, or returning to your list of audits? Does that priority change based on where you are in the process? Can we de-emphasize the lower priority button by making it the outline/secondary style?

**Comment by austinhernandez** [comment]
ok. they now follow this pattern. each section has their own success message and the primary styled button takes you to the "checklist" on the "report home page"

![Image](https://user-images.githubusercontent.com/23194720/210641397-94ccbfb8-f9cd-49b9-b16c-8e9c15000d66.png)


![Image](https://user-images.githubusercontent.com/23194720/210641642-f7928d65-907d-47bf-a28b-cbbbf3c542e0.png)


![Image](https://user-images.githubusercontent.com/23194720/210641528-ffaf3b6e-36da-44d7-8d8f-38e0857c44ae.png)


![Image](https://user-images.githubusercontent.com/23194720/210641857-5021aff1-34d0-4115-803a-a0561644060b.png)


i shortened the text to accommodate the function of a modal.
